mirai-console/backend/mirai-console/src/internal/plugin/PluginManagerImpl.kt

```diff
@@ -180,7 +178,19 @@ internal class PluginManagerImpl(
     }
 
     internal fun enableAllLoadedPlugins() {
-        resolvedPlugins.forEach { enablePlugin(it) }
+        resolvedPlugins.forEach { plugin ->
+            val tmp = plugin.dependencies.firstOrNull { d -> !PluginManager.plugins.first { it.id == d.id }.isEnabled }
+            if (tmp != null) {
+                logger.error("Cannot enable plugin ${plugin.id}, because mirai failed to enable ${tmp.id}")
```

Comment:
```suggestion
                logger.error("Cannot enable plugin '${plugin.id}', because it depends on '${tmp.id}' which was not enabled")
```

插件 `isEnabled` 是 `false` 并不代表它'失败'了, 它也有可能是由某种方式被用户提前手动关闭了